addons/web/tooling/_jsconfig.json

```diff
@@ -25,29 +25,68 @@
             "@pos_sale/*": ["addons/pos_sale/static/src/*"],
+            "@pos_sale_loyalty/*": ["addons/pos_sale_loyalty/static/src/*"],
+            "@pos_sale_margin/*": ["addons/pos_sale_margin/static/src/*"],
             "@pos_sale_product_configurator/*": ["addons/pos_sale_product_configurator/static/src/*"],
+            "@pos_self_order/*": ["addons/pos_self_order/static/src/*"],
+            "@pos_self_order_adyen/*": ["addons/pos_self_order_adyen/static/src/*"],
+            "@pos_self_order_epson_printer/*": ["addons/pos_self_order_epson_printer/static/src/*"],
+            "@pos_self_order_sale/*": ["addons/pos_self_order_sale/static/src/*"],
+            "@pos_self_order_stripe/*": ["addons/pos_self_order_stripe/static/src/*"],
             "@pos_six/*": ["addons/pos_six/static/src/*"],
             "@pos_stripe/*": ["addons/pos_stripe/static/src/*"],
-            "@test_mail/*": ["addons/test_mail/static/src/*"],
-            "@whatsapp_pos/*": ["addons/whatsapp_pos/static/src/*"]
+            "@spreadsheet_dashboard_pos_hr/*": ["addons/spreadsheet_dashboard_pos_hr/static/src/*"],
+
+            "@l10n_be_reports_post_wizard/*": ["l10n_be_reports_post_wizard/static/src/*"],
+            "@l10n_cl_edi_pos/*": ["l10n_cl_edi_pos/static/src/*"],
+            "@l10n_de_pos_cert/*": ["l10n_de_pos_cert/static/src/*"],
+            "@l10n_de_pos_res_cert/*": ["l10n_de_pos_res_cert/static/src/*"],
+            "@l10n_in_reports_gstr_pos/*": ["l10n_in_reports_gstr_pos/static/src/*"],
+            "@l10n_mx_edi_pos/*": ["l10n_mx_edi_pos/static/src/*"],
+            "@l10n_pl_reports_pos_jpk/*": ["l10n_pl_reports_pos_jpk/static/src/*"],
+            "@pos_account_reports/*": ["pos_account_reports/static/src/*"],
+            "@pos_blackbox_be/*": ["pos_blackbox_be/static/src/*"],
+            "@pos_enterprise/*": ["pos_enterprise/static/src/*"],
+            "@pos_hr_mobile/*": ["pos_hr_mobile/static/src/*"],
+            "@pos_iot/*": ["pos_iot/static/src/*"],
+            "@pos_iot_six/*": ["pos_iot_six/static/src/*"],
+            "@pos_l10n_se/*": ["pos_l10n_se/static/src/*"],
+            "@pos_online_payment_self_order_preparation_display/*": ["pos_online_payment_self_order_preparation_display/static/src/*"],
+            "@pos_order_tracking_display/*": ["pos_order_tracking_display/static/src/*"],
+            "@pos_preparation_display/*": ["pos_preparation_display/static/src/*"],
+            "@pos_restaurant_appointment/*": ["pos_restaurant_appointment/static/src/*"],
+            "@pos_restaurant_preparation_display/*": ["pos_restaurant_preparation_display/static/src/*"],
+            "@pos_sale_stock_renting/*": ["pos_sale_stock_renting/static/src/*"],
+            "@pos_self_order_preparation_display/*": ["pos_self_order_preparation_display/static/src/*"],
+            "@pos_settle_due/*": ["pos_settle_due/static/src/*"],
+            "@whatsapp_pos/*": ["whatsapp_pos/static/src/*"],
```

Comment:
<img width="430" alt="Screenshot 2023-11-28 at 14 59 56" src="https://github.com/odoo/odoo/assets/3245568/9f8f25a6-42b3-4ad7-a723-a0b216433e90">

It is originally with `addons`. Did they make a mistake when they declared it here?

Reply:
yes